src/scripts/stats.data.js

```diff
@@ -0,0 +1,46 @@
+const requestOptions = {
+  method: 'GET',
+  redirect: 'follow'
+};
+
+const fetchInfo = (countryName = 'Belarus', mode = 'world') => {
+  let param = countryName.replace(/ /gi, '-').toLowerCase();
+  let data;
+  if (mode === 'world') {

```

Comment:
using if,else you could create a different fetch parametrs and not duplicated code -
`.then((response) => {
      return response.json();
    })
    .catch(error => console.log('error', error));`

Reply:
Thank you for review! I'll correct my part of mistakes.